sqlmesh/core/snapshot/evaluator.py

```diff
@@ -1811,7 +1840,7 @@ def _intervals(snapshot: Snapshot, deployability_index: DeployabilityIndex) -> I
 
 def _check_destructive_schema_change(
     snapshot: Snapshot,
-    alter_expressions: t.List[exp.AlterTable],
+    alter_expressions: t.List[exp.Alter],
```

Comment:
why was this changed? isn't this already in main a while ago? did you not rebase?

Reply:
it's not yet in main #3003